docs/source/intro_accuracy_precision.rst

```diff
@@ -96,6 +100,6 @@ The tools for quantifying DEM precision are described in :ref:`spatialstats`.
     Functions that are used in several examples create duplicate examples intead of being merged into the list.
     Circumventing manually by selecting functions used only once in each example for now.
 
-.. minigallery:: xdem.spatialstats.neff_circ xdem.spatialstats.plot_1d_binning
+.. minigallery:: xdem.spatialstats.infer_heteroscedasticity_from_stable xdem.spatialstats.infer_spatial_correlation_from_stable
```

Comment:
In the final rendered documentation, there are four examples, and the one about spatial correlation of errors appear twice... Any idea why?

Reply:
Yes, this is because I listed two functions and both are used by the spatial propagation examples. So the example is listed twice... If would be good if sphinx-gallery could merge them automatically, but it doesn't. :cry:  
I'll have to come up with something else.

Reply:
I listed spatial stats functions that never appear simultaneously in a single example, not super clean but I don't find anything in sphinx-gallery docs to do this in another manner:
```rst
.. minigallery:: xdem.spatialstats.infer_heteroscedasticity_from_stable xdem.spatialstats.get_variogram_model_func xdem.spatialstats.sample_empirical_variogram
```